cmd/main.go

```diff
@@ -144,6 +137,28 @@ func parseConfig() (Config, error) {
 			retval.ignoreNamespacesRegex = append(retval.ignoreNamespacesRegex, regex)
 		}
 	}
+
+	switch strings.ToLower(retval.ctrlConfig.LogLevel) {
```

Comment:
Can we move this code into a helper function that takes the level and returns a zap.Logger? main() is already pretty big and poorly factored.

Reply:
Where should I create this helper function?